Reformat SVGAnimatedRect (#39345)
mdn/content: files/en-us/web/api/svganimatedrect/index.md

{{APIRef("SVG")}}

> The **`SVGAnimatedRect`** interface correspond to the {{ domxref("SVGRect") }} basic attribute which can be animated.

Reviewer: ```suggestion
The **`SVGAnimatedRect`** interface represents an {{domxref("SVGRect")}} attribute that can be animated.
```
